apps/fishing-map-e2e/src/e2e/map/analysis.cy.ts

```diff
@@ -0,0 +1,54 @@
+import { MAP_POPUP_EEZ_SECTION, SIDEBAR_TOGGLE_EEZ } from '../../constants/buttons'
+import { SEARCH_EEZ, SEARCH_EEZ_FULL_NAME } from '../../constants/search'
+import {
+  disablePopups,
+  getMapCanvas,
+  getSidebar,
+  scrollSidebar,
+  waitForMapLoadTiles,
+  waitForSidebarLoaded,
+} from '../../support/app.po'
+
+describe('See the creation of analysis for an area', () => {
+  before(() => {
+    cy.login(Cypress.env('apiAuthUser'), Cypress.env('apiAuthPass'))
+  })
+  beforeEach(() => {
+    disablePopups()
+    cy.visit('/')
+    waitForSidebarLoaded()
+    waitForMapLoadTiles()
+    scrollSidebar('bottom', 1000)
+    cy.getBySel(SIDEBAR_TOGGLE_EEZ).click()
+    cy.getBySel('map-search-button').click()
+    // I need to add a delay because it doesnt work propetly the autocomplete if we type so fast
```

Comment:
That's probable caused by the debounce that was set to the input text change event.

Reply:
I don't know but I spent lot of time trying to understand why doing it fast was not working